services/common/src/components/reports/ReportDetailsForm-edit.spec.tsx

```diff
@@ -19,6 +19,25 @@ const initialState = {
   },
 };
 
+function mockFunction() {
```

Comment:
does the ReportDetailsForm still need the mock function?

Reply:
Ah, you're right.  That's no longer needed.  With the implementation I had before rebasing it was required, but no longer!